src/clj/prerenderer/core.clj

```diff
@@ -70,10 +74,13 @@
                                                          .deleteOnExit))
                           :start-timeout     5000
                           :wait              false
-                          :noop-when-stopped false}
+                          :noop-when-stopped false
+                          :working-directory "."}
```

Comment:
"." should work fine at least on any bsd/linux/osx/windows. I'd leave it as is and fix if any issues arise, which I doubt would happen.